lisa/features/security_profile.py

```diff
@@ -16,13 +17,13 @@
 @dataclass()
 class SecurityProfileSettings(schema.FeatureSettings):
     type: str = FEATURE_NAME_SECURITY_PROFILE
-    secure_boot_enabled: bool = False
+    security_profile: str = ""
```

Comment:
It's still a string, search DiskType in schema.py. So, other values will be failed. And it needs a default value as "none".